static/js/admin/src/ResultView.js

```diff
@@ -30,14 +30,10 @@ define('ResultView', [ 'jquery', 'jquery-ui', 'ResultObjectList', 'URLUtilities'
 							]
 						},*/
 						{
-							actions : [
-								{action : 'PublishBatch', label : 'Publish'},
-								{action : 'UnpublishBatch', label : 'Unpublish'}
-							]
-						}, {
 							actions : [
 								{action : 'RestoreBatch', label : 'Restore'},
-								{action : 'OpenDeleteBatch', label : 'Delete'}
+								{action : 'OpenDeleteBatch', label : 'Delete'},
```

Comment:
Looking around a bit, it seems that this set of resultActions is passed into ResultObjectActionMenu and used for a bulk actions menu. My recollection is that when multiple things are selected, right clicking to bring up the context menu was supposed to list these actions rather than the usual menu options for individual objects. That doesn't appear to be working here or in production, but the code appears to still be around. See the `batchActions` references in `ResultObjectActionMenu`. 

All that to say, it might be good to check on why that menu isn't working anymore while we're already looking at the bulk actions.

Reply:
Do you think it should be a separate ticket?

Reply:
https://jira.lib.unc.edu/browse/BXC-2688

Reply:
It should work now.